core/tee/tee_fs_private.h

```diff
@@ -43,11 +46,36 @@
 #define TEE_FS_READDIR   12
 #define TEE_FS_RMDIR     13
 #define TEE_FS_ACCESS    14
+#define TEE_FS_LINK      15
+
+#define FILE_BLOCK_SHIFT	4
+
+#define FILE_BLOCK_SIZE		(1 << FILE_BLOCK_SHIFT)
+
+#define NUM_BLOCKS_PER_FILE	1024
+
+#define MAX_FILE_SIZE	(FILE_BLOCK_SIZE * NUM_BLOCKS_PER_FILE)
```

Comment:
Yes, you are right. The value of FILE_BLOCK_SHIFT should be modified to appropriate value, e.g.  19, for 512KB block size, thus max file size could be 512MB. The only reason to have such small block size here is we want to validate cross-block operation by x-test, which usually write small number of bytes to TEE file.